File: src/main/java/net/botlify/anticaptchacom/response/solution/ImageToTextSolution.java
package net.botlify.anticaptchacom.response.solution;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.ToString;

@Getter
@ToString
@EqualsAndHashCode
public class ImageToTextSolution {

  /**
   * Text from image captcha.
   */
  @JsonProperty("text")
  private String text;

  /**
   * Web address where we'll store the captcha for the next 24 hours, after which it is removed.
   */
  @JsonProperty("url")
  private String url;

}

File: src/main/kotlin/net/botlify/anticaptchacom/data/response/solution/ImageToTextSolution.kt
package net.botlify.anticaptchacom.data.response.solution

import com.fasterxml.jackson.annotation.JsonProperty

@Deprecated("Not implemented yet")
data class ImageToTextSolution(
    /**
     * Text from image captcha.
     */
    @JsonProperty("text")
    private val text: String,

    /**
     * Web address where we'll store the captcha for the next 24 hours, after which it is removed.
     */
    @JsonProperty("url")
    private val url: String
)
